### PR TITLE
Stats: Add banner for Parse.ly

### PR DESCRIPTION
**Pull request by sixhours**
#### Changes proposed in this Pull Request

* Replace Cloudflare and money upsells with Parse.ly banner

**Before**

wordpress com_stats_day_sixhours wpcomstaging com

**After**

calypso localhost_3000_stats_day_sixhours wpcomstaging com

#### Testing instructions

* Switch to this PR
* Navigate to `/stats/[site]`
* Scroll down - there should be a Parse.ly banner at the bottom of the screen
* Make sure "Learn more" links to `http://parse.ly`
* Check to make sure Tracks events fire when you view and click on the banner - all prefixed with `calypso_stats_parsely_banner_*`

### CODE REVIEW

File: client/my-sites/stats/site.jsx [hunk 1]
```diff
@@ -300,28 +286,21 @@ class StatsSite extends Component {
...
+				<Banner
+					callToAction={ translate( 'Learn more' ) }
+					description={ translate(
+						"Parse.ly Analytics makes it easy to understand the full impact of your content. Measure what's driving awareness, engagement, and conversions."
+					) }
+					dismissPreferenceName="calypso_stats_parsely_banner_dismiss"
+					event="calypso_stats_parsely_banner_view"
+					href="http://parse.ly"
+					iconPath={ parselyIcon }
+					title={ translate( 'Discover more stats with Parse.ly Analytics' ) }
```
Comment (a8ci18n) [inline client/my-sites/stats/site.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Discover more stats with Cloudflare Analytics' )` *ES Score: 6*

**Comment by olaolusoga** [comment]
Looking good. Thanks for moving fast on this.

Notes:
- increase card margin-top, and all padding to 16px
- Increase margin-right for banner-icon to 12px

**Comment by sixhours** [comment]
> increase card margin-top, and all padding to 16px
> Increase margin-right for banner-icon to 12px

Sure thing! All done :)

**Comment by mreishus** [comment]
I believe there are more unused props that can be removed from connect: [isAdmin](https://github.com/Automattic/wp-calypso/blob/402bc83e8f70db5cb26bfef8e161181857bc4483/client/my-sites/stats/site.jsx#L374), [isVip](https://github.com/Automattic/wp-calypso/blob/402bc83e8f70db5cb26bfef8e161181857bc4483/client/my-sites/stats/site.jsx#L377), [planSlug](https://github.com/Automattic/wp-calypso/blob/402bc83e8f70db5cb26bfef8e161181857bc4483/client/my-sites/stats/site.jsx#L379) (not a result of these changes but still unused), [planSupportsWordAdsInstantFeature](https://github.com/Automattic/wp-calypso/blob/402bc83e8f70db5cb26bfef8e161181857bc4483/client/my-sites/stats/site.jsx#L380).

**Comment by sixhours** [comment]
We're going to swap out the Banner component for something like PromoCard to make this look a little bolder. Putting this on hold until we have the design ready.

File: client/my-sites/stats/site.jsx [hunk 2]
```diff
@@ -300,28 +283,27 @@ class StatsSite extends Component {
...
-							href={ `/earn/${ slug }` }
-							dismissPreferenceName={
-								planSupportsWordAdsInstantFeature ? `stats-earn-nudge-wordads-${ siteId }` : null
+				</div>
+				<Banner
+					className="stats__parsely-banner"
+					callToAction={ translate( 'Learn more' ) }
+					description={ preventWidows(
+						translate(
+							"Parse.ly Analytics makes it easy to understand the full impact of your content. {{br/}} Measure what's driving awareness, engagement, and conversions.",
```
Comment (obenland) [inline client/my-sites/stats/site.jsx]: Could we start the description with "Need deeper insights?"?

**Comment by simison** [comment]
Just curious, will the earn banner get removed with this PR? 

image

**Edit**: doh, says so right in the PR description. 👍 

> and money upsells

File: client/my-sites/stats/site.jsx [hunk 3]
```diff
@@ -300,28 +288,36 @@ class StatsSite extends Component {
...
+				</div>
+				<Card className="stats__parsely-banner">
+					<TrackComponentView eventName="calypso_stats_parsely_banner_view" />
+					<img src={ parselyIcon } alt="" />
+					<div>
+						<FormattedHeader
+							brandFont
+							className="stats__parsely-banner-header"
+							headerText={ preventWidows(
+								translate( 'Discover more stats with Parse.ly Analytics' )
```
Comment (a8ci18n) [inline client/my-sites/stats/site.jsx:300]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Discover more stats with Cloudflare Analytics' )` *ES Score: 6*

File: client/my-sites/stats/site.jsx [hunk 4]
```diff
@@ -300,28 +288,36 @@ class StatsSite extends Component {
...
+									{
+										components: {
+											br: <br />,
+										},
+									}
+								)
+							) }
+						</p>
+					</div>
+					<Button primary href="https://parse.ly" onClick={ this.parselyClick }>
```
Comment (alshakero) [inline client/my-sites/stats/site.jsx]: Optional: as a user I'd expect the Learn More button to open a new tab.

File: client/my-sites/stats/site.jsx [hunk 5]
```diff
@@ -300,28 +288,36 @@ class StatsSite extends Component {
...
-						<Banner
-							className="stats__upsell-nudge"
-							icon="star"
-							title={ translate( 'Start earning money now' ) }
-							description={ translate(
-								'Accept payments for just about anything and turn your website into a reliable source of income with payments and ads.'
+				</div>
+				<Card className="stats__parsely-banner">
+					<TrackComponentView eventName="calypso_stats_parsely_banner_view" />
+					<img src={ parselyIcon } alt="" />
```
Comment (alshakero) [inline client/my-sites/stats/site.jsx]: Maybe give this `aria-hidden="true"` since it's only aesthetic and doesn't have an alt?

**Review by alshakero — APPROVED** [review]
Works great! Left two optional comments.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6570627

Thank you @sixhours for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6570627).

**Comment by tyxla** [comment]
Removed the unused Cloudflare banner in #60126.

**Pull request merged**